platform/darwin/src/MGLRendererConfiguration.h

```diff
@@ -26,11 +26,11 @@ MGL_EXPORT
 /** The name of the font family to use for client-side text rendering.
 
  Currently only used for CJK glyphs. Changing this at run time is not currently
- supported. Enable client-side rendering of CJK glyphs by setting
- `MGLIdeographicFontFamilyName` in your containing app's Info.plist to a value
- which will be available at run time. Default font for local ideograph font family
- is "PingFang". */
-@property (nonatomic, readonly) std::string localFontFamilyName;
+ supported. Enable client-side rendering of CJK glyphs and use `Helvetica` by default.
+ Set `MGLIdeographicFontFamilyName` in your containing app's Info.plist to a value
+ which will be available at run time, e.g. "PingFang".
+ Set `MGLIdeographicFontFamilyName` to `NULL` for using your custom remote font.*/
```

Comment:
The default font is `PingFang` but not the `Helvetica`.
In [MGLRendererConfiguration.mm](https://github.com/mapbox/mapbox-gl-native/blob/77c8b1952c6a0935ff2ff74481a3285c61930936/platform/darwin/src/MGLRendererConfiguration.mm), we use system font as following:
```
[[UIFont systemFontOfSize:0 weight:UIFontWeightRegular].familyName UTF8String]
```

Since we only support iOS9+, so this system default font is `.SFUIText`, and for kanji characters, it's `PingFang`.

and fyi, in `local_glyph_rasterizer.mm`, we generate font as following:
```
    CTFontRef getFont() {
        if (!fontFamily) {
            return NULL;
        }
        
        if (!fontHandle) {
          NSDictionary *fontAttributes = @{
                (NSString *)kCTFontSizeAttribute: [NSNumber numberWithFloat:24.0],
                (NSString *)kCTFontFamilyNameAttribute: [[NSString alloc] initWithCString:fontFamily->c_str() encoding:NSUTF8StringEncoding]
            };

            CTFontDescriptorRefHandle descriptor(CTFontDescriptorCreateWithAttributes((CFDictionaryRef)fontAttributes));
            fontHandle = CTFontCreateWithFontDescriptor(*descriptor, 0.0, NULL);
            if (!fontHandle) {
                throw std::runtime_error("CTFontCreateWithFontDescriptor failed");
            }
        }
        return fontHandle;
    }
```
If the `fontFamily` string is unrecognized, it will fallback to `Helvetica` by default.